BibTeX.js

```diff
@@ -262,7 +263,14 @@ function setKeywordDelimRe( val, flags ) {
 function processField(item, field, value, rawValue) {
 	if(Zotero.Utilities.trim(value) == '') return null;
 	if(fieldMap[field]) {
-		item[fieldMap[field]] = value;
+		//map DOIs + Label to Extra for unsupported item types
```

Comment:
I'll leave this here -- I'd like to leave the DOI in the generic field mapping (l. 84) and the only way to get to `else` would be to remove it there.